.travis.yml

```diff
@@ -89,9 +89,11 @@ jobs:
         - yes | sdkmanager "platforms;android-27" # see https://github.com/travis-ci/travis-ci/issues/8874#issuecomment-350350607
         - scripts/decrypt.sh
         - scripts/signing.sh
-        - curl -sL https://deb.nodesource.com/setup_10.x | sudo -E bash -
+        - curl -sL https://deb.nodesource.com/setup_12.x | sudo -E bash -
```

Comment:
Without this change, travis was installing node 10.14.0, but one of our deps (pretty-format) requires >= 1.14.2. I did not find a way to use a different 10;x version, so switched to the nest LTE. In theory this has no consequence but I don't like that we have to use 2 different versions of node. Should we also upgrade the web version?

Reply:
> I don't know why we don't have to upgrade the web version too?

The web version uses a node image from travis, which installs node 10.20.0 and satisfies the requirement from pretty-format.
[Here's the PR](https://github.com/facebook/jest/pull/9423) — the change is from 2 weeks ago.

> Also which package update is responsible for the pretty-format update?

I can check this, but it's a jest package. I don't like our chances of being able to downgrade it.

We use [the recommended way to install node on debian](https://github.com/nodesource/distributions/blob/master/README.md) but I don't see an option to change the version.
We could also switch to node 12 for the web project?

Reply:
> The web version uses a node image from travis, which installs node 10.20.0 and satisfies the requirement from pretty-format.

Why not using the same thing so? 

I don't think we want to upgrade to node 12 yet.

Reply:
> Why not using the same thing so?

We're using the `android` image of travis at the moment… which makes sense for building an android image 😁. Otherwise we'd have to install a lot of dependencies and build tools manually.

> I don't think we want to upgrade to node 12 yet.

Why not though? The only issue I see is that the other apps still use node 10 and not all of them can be upgraded straight away.

Reply:
I'm not very confident about this node 12 upgrade for now, but let's use that since it's just here to build dev APK. We can test it like that